_dashboards/visualize/area.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Using Area charts
+parent: Visualization types
+grand_parent: Visualize
+nav_order: 5
+---
+
+# Using Area charts
+
+An area chart is a line chart with the area between the line and the axis shaded with a color, and is a primary visualization type used to display time series data. You can create area charts in Dashboards using the Area visualization type or using the TSVB, Vega, or VisBuilder visualization tools. For this tutorial, you'll use the Area visualization type.
+
+![Demonstration of the area chart tutorial steps]({{site.url}}{{site.baseurl}}/images/dashboards/area-tutorial.gif)
+
+# Try it: Create a simple aggregation-based area chart
+
+In this tutorial you'll create a simple area chart using sample data and aggregations in OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser.
+
+You have several aggregation options in Dashboards, and the choice influences your analysis. The use cases of aggregations vary from analyzing data in real time to using Dashboards to create a visualization dashboard. If you need an overview of aggregations in OpenSearch, see [Aggregations]({{site.url}}{{site.baseurl}}/opensearch/aggregations/) before getting started with this tutorial.
+
+Make sure you have [installed the latest version of Dashboards](https://opensearch.org/docs/latest/install-and-configure/install-dashboards/index/) and added the sample data before continuing with this tutorial. _This tutorial is uses Dashboards version 2.4.1, which is the latest version at time of this tutorial_.
+{: .note}
+
+## Set up the area chart
+
+1. Access Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser.
+1. Select **Visualize** from the menu and then select **Create visualization**.
+1. Select **Area** from the window.
+1. Select **opensearch_dashboards_sample_data_flights** in the **New Area/Choose a source** window.
+1. Select the calendar icon and set the time filter to **Last 7 days.**
+1. Select **Update**.
+
+## Add aggregations to the area chart
+
+Continuing with the area chart created in the preceding steps, you'll create an area chart that displays the top five logs for flights delayed for every three hours over the last seven days.
```

Comment:
```suggestion
Continuing with the area chart created in the preceding steps, you'll create an area chart that displays the top five logs for flights delayed for every three hours over the last seven days:
```

Reply:
The end of this sentence is slightly confusing. The reader will likely expect "by" to follow "delayed", so the meaning of "for every three hours" is a bit unclear.